### PR TITLE
Hardcoded v1 of profile's about me section

### PR DESCRIPTION
**Pull request by rupalg**
Created "about me" section of the profile with hardcoded data. 

Ensured that formatting matched the layout on Ashley's implementation of the "how I can help section": https://github.com/micromentor-team/mm-flutter-app/pull/83

Currently, the country icon and the language icon doesn't match the Figma mocks. Is this necessary to match? There are no country icons on Flutter Icons widget.

Screenshot 2023-08-08 at 3 11 16 PM

### CODE REVIEW

File: lib/widgets/screens/profile/profile_about_me.dart [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        ]);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final AppLocalizations l10n = AppLocalizations.of(context)!;
+    final ThemeData theme = Theme.of(context);
+
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.start,
```
Comment (jupblb) [inline lib/widgets/screens/profile/profile_about_me.dart:83]: You don't need to set this value to `MainAxisAlignment.start` as it's the default one. This applies to other parts of this PR where it's used.

File: lib/widgets/screens/profile/profile_about_me.dart [hunk 2]
```diff
@@ -0,0 +1,156 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+import '../../../constants/app_constants.dart';
+import '../../atoms/profile_chip.dart';
+
+class ProfileAboutMe extends StatelessWidget {
+  const ProfileAboutMe({Key? key}) : super(key: key);
```
Comment (jupblb) [inline lib/widgets/screens/profile/profile_about_me.dart:8]: Nit: this may be simplified to `const ProfileAboutMe({super.key});`

**Review by jupblb — CHANGES_REQUESTED** [review]
image
The chip sections is slightly misaligned when compared with the wireframe. Once https://github.com/micromentor-team/mm-flutter-app/pull/89 is merged you may want to use the `ProfileChipPadded` widget.

File: lib/widgets/screens/profile/profile_about_me.dart [hunk 3]
```diff
@@ -0,0 +1,156 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+import '../../../constants/app_constants.dart';
```
Comment (jupblb) [inline lib/widgets/screens/profile/profile_about_me.dart:4]: Nit: it seems to me we are mostly using absolute imports throughout the app (`package:mm_flutter_app/...`).

File: lib/widgets/screens/profile/profile_about_me.dart (hunk 1 above)
Comment (rupalg) [reply lib/widgets/screens/profile/profile_about_me.dart:83]: Hi, okay- will remove this value wherever applied

File: lib/widgets/screens/profile/profile_about_me.dart (hunk 2 above)
Comment (rupalg) [reply lib/widgets/screens/profile/profile_about_me.dart:7]: okay

File: lib/widgets/screens/profile/profile_about_me.dart (hunk 3 above)
Comment (rupalg) [reply lib/widgets/screens/profile/profile_about_me.dart:3]: Good point. I am happy to keep the path absolute. I am seeing a combination of relative and absolute imports throughout the app though, so for future PRs, we can keep the path absolute.

**Pull request closed without merging**